pbs/usersync.go

```diff
@@ -53,7 +57,7 @@ type UserSyncMap interface {
 	Unsync(familyName string)
 	// TrySync tries to set the UID for some family name. It returns an error if the set didn't happen.
 	TrySync(familyName string, uid string) error
-	// HasSync returns true if we have a UID for the given family, and false otherwise.
+	// HasSync returns true if we have an active UID for the given family, and false otherwise.
 	HasSync(familyName string) bool
 	// SyncCount returns the number of families which have UIDs for this user.
 	SyncCount() int
```

Comment:
we've gotta get you on a real IDE :).

The uses are [here](https://github.com/prebid/prebid-server/pull/86/files#diff-444feda808b76687bcc8827c1025876cR183) and [here](https://github.com/prebid/prebid-server/pull/86/files#diff-444feda808b76687bcc8827c1025876cR232).

I didn't actually look too closely at what they were doing... just needed to translate the existing code when I made the UIDs private.